src/components/StarRating/StarRating.scss

```diff
@@ -0,0 +1,31 @@
+$color-placeholder-star: #e1e1e1;
+$color-active-star: #d9af00;
+$color-number-reviews: gray;
+
+.CoveoStarRating {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+}
+
+.coveo-star-rating-star + .coveo-star-rating-star {
```

Comment:
Why are you using the adjacent selector on a class with itself?

Reply:
That's actually smart but a costly selector in CSS :) don't know what we want to do with it.